packages/account/src/Components/icon-message-list/index.ts

```diff
@@ -0,0 +1,3 @@
+import IconMessageContent from './icon-message-list';
```

Comment:
Please verify the name. There is a icon-message-component seperately. Can we rename this as IconMessageList to prevent confusion